<ui version="4.0" >
 <class>DialogIcepapProgram</class>
 <widget class="QDialog" name="DialogIcepapProgram" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>239</height>
   </rect>
  </property>
  <property name="palette" >
   <palette>
    <active>
     <colorrole role="Highlight" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>101</red>
        <green>148</green>
        <blue>235</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Highlight" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>101</red>
        <green>148</green>
        <blue>235</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Highlight" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>127</red>
        <green>125</green>
        <blue>123</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle" >
   <string>Icepap Program</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="icepapcms.qrc" >:/icons/IcepapCfg Icons/gnome-cpu.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <property name="horizontalSpacing" >
    <number>6</number>
   </property>
   <property name="verticalSpacing" >
    <number>6</number>
   </property>
   <item row="2" column="6" colspan="2" >
    <widget class="QLineEdit" name="txtHost" />
   </item>
   <item row="1" column="1" >
    <widget class="QComboBox" name="cbProgram" >
     <item>
      <property name="text" >
       <string>NONE</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>ALL</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>DRIVERS</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>COMM</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>ADDR</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="5" colspan="3" >
    <widget class="QPushButton" name="btnProgram" >
     <property name="text" >
      <string>Upgrade firmware</string>
     </property>
     <property name="icon" >
      <iconset resource="icepapcms.qrc" >:/icons/IcepapCfg Icons/gnome-cpu.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="4" colspan="2" >
    <widget class="QRadioButton" name="rbSerial" >
     <property name="text" >
      <string>Serial Line</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Firmware file</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6" colspan="2" >
    <widget class="QComboBox" name="cbSerial" />
   </item>
   <item row="6" column="5" colspan="3" >
    <widget class="QPushButton" name="btnClose" >
     <property name="text" >
      <string>OK</string>
     </property>
     <property name="icon" >
      <iconset resource="icepapcms.qrc" >:/icons/IcepapCfg Icons/close.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Program</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7" >
    <widget class="QToolButton" name="btnBrowser" >
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5" colspan="3" >
    <widget class="QPushButton" name="btnTest" >
     <property name="text" >
      <string>Test connection</string>
     </property>
     <property name="icon" >
      <iconset resource="icepapcms.qrc" >:/icons/IcepapCfg Icons/gnome-nettool.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="4" colspan="2" >
    <widget class="QRadioButton" name="rbEth" >
     <property name="text" >
      <string>Ethernet</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Options</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="6" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2" >
    <widget class="QSpinBox" name="sbAddr" >
     <property name="minimum" >
      <number>1</number>
     </property>
     <property name="maximum" >
      <number>128</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QComboBox" name="cbOptions" >
     <item>
      <property name="text" >
       <string>SAVE</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>SL</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>NONE</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" colspan="6" >
    <widget class="QLineEdit" name="txtFirmwareFile" >
     <property name="maximumSize" >
      <size>
       <width>1666666</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item rowspan="4" row="3" column="0" colspan="4" >
    <widget class="QTextEdit" name="txtLog" >
     <property name="font" >
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QCheckBox" name="chkForce" >
     <property name="text" >
      <string>Force</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtFirmwareFile</tabstop>
  <tabstop>btnBrowser</tabstop>
  <tabstop>cbProgram</tabstop>
  <tabstop>sbAddr</tabstop>
  <tabstop>cbOptions</tabstop>
  <tabstop>rbSerial</tabstop>
  <tabstop>rbEth</tabstop>
  <tabstop>cbSerial</tabstop>
  <tabstop>txtHost</tabstop>
  <tabstop>btnTest</tabstop>
  <tabstop>btnProgram</tabstop>
  <tabstop>btnClose</tabstop>
  <tabstop>txtLog</tabstop>
 </tabstops>
 <resources>
  <include location="icepapcms.qrc" />
 </resources>
 <connections/>
</ui>
